content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Deploying to Vercel'
 metaTitle: 'How to deploy a Prisma app to Vercel'
-metaDescription: 'Learn how to deploy a Next.js app with a serverless REST API using Prisma Client to Vercel.'
+metaDescription: 'Learn how to deploy a Next.js app with a serverless REST API, with Prisma Client to Vercel.'
```

Comment:
I see the point of avoiding 'using', but this reads very oddly to me and took me a few goes to make sense of... sounds like 'Prisma Client to Vercel' is a product

Maybe 'Learn how to use Prisma Client to...' instead?

Reply:
Good call!